Makefile

```diff
@@ -1,3 +1,6 @@
+# Alllow developer to override some defaults
+-include devel.mk
```

Comment:
Great! I learned something today. (Dash prefix to ignore errors when the files doesn't exist)